src/components/member-profile/contribution/index.js

```diff
@@ -167,6 +181,37 @@ const ContributionCard = ({
     </div>
   );
 };
+const ContributionCollapsedCard = ({ contribution }) => {
```

Comment:
why is this component declared in this same file ?

Reply:
done